.gitignore

```diff
@@ -17,3 +17,5 @@ tests/freetype/objs/*.lo
 third_party/lzma.js/lzip/*.o
 third_party/lzma.js/lzma-native
 third_party/lzma.js/lzma-native.exe
+
```

Comment:
I'm not sure why emscripten would emit anything to that location. Did you tell it to? Or maybe it was the SDK? cc @juj